js/okx.js

```diff
@@ -2387,6 +2387,24 @@ module.exports = class okx extends Exchange {
         return this.parseOrders (data, market, since, limit);
     }
 
+    async fetchClosedOrder (id, symbol = undefined, params = {}) {
```

Comment:
We should not emulate this endpoint with `fetchClosedOrders` if the underlying exchange does not have a direct endpoint for fetching a single order.